core/src/main/java/org/apache/accumulo/core/client/rfile/RFileScanner.java

```diff
@@ -218,7 +218,7 @@ public void indexWeightChanged() {}
     if (null == this.dataCache) {
       this.dataCache = new NoopCache();
     }
-    this.cryptoService = CryptoServiceFactory.getConfigured(tableConf);
+    this.cryptoService = CryptoServiceFactory.newInstance(tableConf);
```

Comment:
Not sure if tableConf can be used here.  Will it have your crypto configuration?

Reply:
The impl of RFile.withTableProperties() will just take whatever key values are passed and put them in a map.  It is almost as if this method should be renamed to "withProperties" since we allow more than just table properties.  Since it is already released as public API maybe we can just elaborate more in the javadoc.  I will create a follow on issue to investigate this.  

I don't think there is anything I can do for this PR though since we don't want to put the crypto stuff in the public API while its still experimental.

Reply:
Created #611